content/en/api/logs/configuration

```diff
@@ -0,0 +1,376 @@
+  filter: {
+    query: "source:java"       // A query string. For search query syntax, see:
+                               // https://docs.datadoghq.com/logs/explorer/search/
+  }
+
+  // Ordered list of child processors and nested pipelines (see below for definition)
+  processors: [
+    {
+      name: 'Parsing Java Default formats'  // Name of the processor, for display only.
+      type: 'grok-parser'                   // Type of processor. See below for options.
+      is_enabled: true                      // True if this processor is enabled and processing logs.
+      // For the many other Processor-specific properties, see below
+    }
+    {
+      name: 'Define timestamp as the official timestamp of the log'
+      type: 'date-remapper'
+      ...
+    }
+    {
+      name: 'Define level as the official status of the log'
+      type: 'status-remapper'
+      ...
+    }
+  ]
+}
+```
+
+`processors`
+
+```
+type: attribute-remapper
+name: ''
+enabled: true
+sourceType: attribute // (Optional) defaulted to `attribute`, defines if the sources are from log attributes or tags
+sources: // list of source, can be a list of attributes or tags, the first existing one is used  
+  - dyno
+target: 'dyno' // Final attribute or tag name for the first source element that exist  
+targetType: 'tag' // (Optional) defaulted to `attribute`, define if the target is a log attribute or a tag  
+preserveSource: false // `true` or `false`, remove or preserve the remapped source element 
+overrideOnConflict: false // `true` or `false`, override or not the target element if already set
+```
+
+
+```
+type: url-parser
+name: ''
+enabled: true
+sources: // list of sources attribute, the first existing one is used    
+  - http.url
+target: http.url_details //  Name of the parent attribute that will contain all the extracted details from the source 
+```
+
+```
+type: user-agent-parser
+name: ''
+enabled: true
+sources: // list of sources attribute, the first existing one is used
+  - http.useragent
+target: http.useragent_details // Name of the parent attribute that will contain all the extracted details from the source  
+encoded: false // `true` or `false`, define if the source attribute is url encoded or not
+```
+
+```
+type: category-processor
+name: ''
+enabled: true
+categories: // list of filter and corresponding name    
+  - filter:
+      query: '@http.status_code:[200 TO 299]'
+    name: OK
+  - filter:
+      query: '@http.status_code:[300 TO 399]'
+    name: notice
+  - filter:
+      query: '@http.status_code:[400 TO 499]'
+    name: warning
+  - filter:
+      query: '@http.status_code:[500 TO 599]'
+    name: error
+target: http.status_category // Name of the target attribute which value is defined by the matching category
+```
+
+```
+type: grok-parser
+name: ''
+enabled: true
+source: message // name of the log attribute that is parsed (defaulted to `message`)
+grok: list of SupportRules (optional) and Match rules  
```

Comment:
@NBParis  what is s SupportRule ? I cannot find in user documentation

Reply:
@pcariou-dd those are the helper rules in the advanced settings of the grok parser. [Link to documentation](https://docs.datadoghq.com/logs/processing/parsing/?tab=matcher#advanced-settings).